app/graphql/types/query_type.rb

```diff
@@ -31,4 +31,20 @@ def events(after: nil, type: nil)
     events = events.with_source(type) if type
     events
   end
+
+  field :employees, [Types::EmployeesType] do
+    description "Employee events from bambooHR"
+    argument :type, String, required: false
+  end
+
+  def employees(type: nil)
+    puts type
+    puts Clients::BamboohrClient.find_employees.find_birthday_employees()
+    case type
+    when 'Anniversary' then Clients::BamboohrClient.find_employees.find_anniversary_employees()
+    when 'Birthday' then Clients::BamboohrClient.find_employees.find_birthday_employees()
+    when 'TimeOff' then Clients::BamboohrClient.find_employees.find_time_off_employees()
+    else Clients::BamboohrClient.find_employees()
```

Comment:
This should probably be `when nil then ...`. If the user mistypes the `type` argument (e.g. `Timeoff` instead of `TimeOff`) it's better to produce an error rather than to show them incorrect data.

Reply:
I'd recommend revising the schema here. We have a bit of a naming collision with `events`. Maybe that could renamed to something more specific like `collaboration_events` and your new field could be `employee_events`. `employee_events` could have fields `anniversary`, `birthday`, `time_off`. This makes querying a little easier since the fields are discoverable and removes errors either returning a full list of employees e.g. if we had a typo `Aniversary` in the frontend it might appear to be working.

Have a look at the current `events` field and `EventCollectionType` to see how this can be done.